README.md

```diff
@@ -5,11 +5,25 @@ SonarQube Scanner for Jenkins
 [![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/sonar.svg)](https://plugins.jenkins.io/sonar)
 [![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/sonar.svg?color=blue)](https://plugins.jenkins.io/sonar)
 
-This plugin allows easy integration in Jenkins projects of [SonarQube](https://www.sonarsource.com/products/sonarqube/), the open-source solution for helping developers write [Clean Code](https://www.sonarsource.com/solutions/clean-code/?utm_medium=referral&utm_source=github&utm_campaign=clean-code&utm_content=sonar-scanner-jenkins).
+This plugin allows easy integration in Jenkins projects of SonarQube ([Server](https://www.sonarsource.com/products/sonarqube/), [Cloud](https://www.sonarsource.com/products/sonarcloud/)) and [SonarQube Community Build](https://www.sonarsource.com/open-source-editions/sonarqube-community-edition/).
+
+<picture>

```

Comment:
Added the logo to be consistent with the other 'extensions'
Crosschecked with Rachel and the best is using the generic logo, and referring to the specific products in the text.

The logo is different depending on the mode.